packages/codelists/src/components/codelist-detail/view-container.js

```diff
@@ -19,19 +37,37 @@ const CodelistComponentView = (props) => {
 
 	useEffect(() => {
 		API.getDetailedCodelist(id)
-			.then((codelist) => {
-				if (codelist.codes) {
-					codelist.codes = Object.values(codelist.codes).reduce((acc, c) => {
-						return {
-							...acc,
-							[c.code]: {
-								...c,
-								id: c.code,
-							},
-						};
-					}, {});
+			.then((cl) => {
+				if (cl.codes) {
+					cl.codes = Object.values(cl.codes)
```

Comment:
You should create a dedicated method for this code, because I think it is duplicated. 
You can create an `formatCodeList()` method inside a utils.js file for example
Once this method is available, you will be able to call it like : 

```javascript
API.getDetailedCodelist(id).then((cl) => formatCodeList(cl));
```